using_openshift/generate_cmd.adoc

```diff
@@ -0,0 +1,107 @@
+= Command Line Interface
```

Comment:
Just noticed this, which I assume was just a copy/paste remnant. Even though the `Name:` value in `_build_cfg.yml` seems to determine the topic title that is actually rendered in builds, this line might as well be updated to match (e.g., "Config Generation").